devito/ops/transformer.py

```diff
@@ -1,3 +1,38 @@
+from devito.ir.iet import Expression, FindNodes
+from devito.ops.node_factory import OPSNodeFactory
+from devito.ops.nodes import OPSKernel
+from devito.ops.utils import namespace
+
+
+def opsit(trees, count):
+    node_factory = OPSNodeFactory()
+    expressions = []
+    for tree in trees:
+        expressions.extend([
+            Expression(make_ops_ast(i.expr, node_factory))
+            for i in FindNodes(Expression).visit(tree.inner)
+        ])
+
+    arguments = set()
+    to_remove = []
+    for exp in expressions:
+        func = [f for f in exp.functions if f.name != "OPS_ACC_size"]
```

Comment:
this looks hacky?

Reply:
do you have a suggestion on how to do this properly?

Reply:
Add a global tuple of "private names" and check if it's in there?

Alternatively, what is ``f`` when its name is "OPS_ACC_size" ?